src/main/scala/org/broadinstitute/dsde/workbench/sam/api/UserInfoDirectives.scala

```diff
@@ -22,11 +22,30 @@ trait UserInfoDirectives {
   val directoryDAO: DirectoryDAO
```

Comment:
Rename trait to `SamUserDirectives`?

Reply:
I am going to delay this one until the next PR